media/js/firefox/new2.js

```diff
@@ -0,0 +1,280 @@
+                // still need to use the regular trackAndRedirect() function because
+                // the popup will be blocked and then the download will also be blocked
+                // in the popup.
+                if (window.location.hash === '#download-fx' || params.get('scene') === 2) {
+                    trackAndRedirect(url, virtualUrl);
+                } else {
+                    trackAndPopup(url, virtualUrl);
+                }
+            } else {
+                trackAndRedirect(url, virtualUrl);
+            }
+
+            if ($downloadInteraction.data('scene') !== 2) {
+                if (hashChange) {
+                    window.location.hash = '#download-fx';
+                } else {
+                    showScene(2);
+                }
+            }
+        });
+
+        if (hashChange && !noScene2) {
+            $window.on('hashchange', function() {
+                if (window.location.hash === '#download-fx') {
+                    showScene(2);
+                }
+                if (window.location.hash === '') {
+                    showScene(1);
+                }
+            });
+        }
+
+        $window.on('load', function() {
+            // initiate download/scene2 if coming directly to #download-fx and/or ?scene=2
+            // some older browsers will not preserve the #download-fx when they are redirected
+            // so we use a url parameter, but we don't want to use a url param when the user
+            // clicks the download button on /firefox/new/ because that can trigger an unecessary page load
+            if (window.location.hash === '#download-fx' || params.get('scene') === 2) {
+                if (noScene2) {
+                    // if using an unsupported platform just try to drop the URL hash
+                    if (window.history && window.history.replaceState) {
+                        var uri = window.location.href.split('#')[0];
+                        window.history.replaceState({}, '', uri);
+                    }
+                } else {
+                    showScene(2);
+                    // For IE < 11 we supress the auto-download since this
+                    // will soon be triggered using a popup on bedrock prior
+                    // to landing on /firefox/download/#download-fx. This
+                    // check reflects the current logic on bedrock and the old
+                    // PHP download pages (which also only check for `MSIE`).
+                    // Once all buttons point to this page on all locales,
+                    // we can switch this check to IE < 9.
+                    if (navigator.appVersion.indexOf('MSIE') !== -1) {
+                        return;
+                    }
+
+                    $directDownloadLink.trigger('click');
+                }
+            }
+
+            // prep both scenes for display
+            $downloadStarted.removeClass('invisible');
```

Comment:
Hm, that is a good thought. They would see the headline and logo and such, but not the main content. I could make scene 1 visible by default, then fade in scene 2 on `window.load` if instructed by URL params.

This catch is that it's possible the download button would be visible but then disappear as the user is reaching for it. However, the download would start automatically in this case (on IE 9+, at least). Probably worth it?